_posts/2016-07-29-live-streamed-demos-for-exponential-transparency-and-information-sharing.md

```diff
@@ -50,15 +50,15 @@ work.
 
 -   YouTube is available to federal employees in most agencies, while also allowing us to have multiple presenters share their screen and demo with audio and video.
 -   After the live stream finishes streaming, the video becomes available on-demand for playback, like any YouTube video, so people can watch later if they missed the live session, re-watch, and share with their colleagues. This allowed us to do the demo for dozens of stakeholders, potential clients, and regulations fanatics who helped us guide the product development process.
--   We figured out a way to do use the live stream option, but also allow for some interactivity. While streaming, there is about a 30-second delay between when the presenters are presenting and when it shows up on the live stream, so to manage that, we split the demos into two parts — a presentation and then a forum/feedback session — in which we use a phone conference line on a speakerphone to record the discussion into the video for questions and answers back and forth without the 30-second latency. If we need to, we can still use the visuals of Youtube (with the 30-second delay).
+-   We figured out a way to use the live stream option, but also allow for some interactivity. While streaming, there is about a 30-second delay between when the presenters are presenting and when it shows up on the live stream, so to manage that, we split the demos into two parts — a presentation and then a forum/feedback session — in which we use a phone conference line on a speakerphone to record the discussion into the video for questions and answers back and forth without the 30-second latency. If we need to, we can still use the visuals of Youtube (with the 30-second delay).
```

Comment:
It's hard to tell from this diff, but I changed the beginning of this paragraph to "We figured out a way to ~~do~~ use".